pom.xml

```diff
@@ -21,265 +27,56 @@
         <url>https://github.com/elucidation-project</url>
     </organization>
 
-    <developers>
-        <developer>
-            <name>Chris Rohr</name>
-            <url>https://gitlab.com/chrisrohr</url>
-        </developer>
-        <developer>
-            <name>Terez Ivy</name>
-            <organization>Fortitude Technologies, LLC</organization>
-            <organizationUrl>https://gitlab.com/fortitudetec</organizationUrl>
-            <url>https://gitlab.com/Terezivy</url>
-        </developer>
-        <developer>
-            <name>Scott Leberknight</name>
-            <organization>Fortitude Technologies, LLC</organization>
-            <organizationUrl>https://gitlab.com/fortitudetec</organizationUrl>
-            <url>https://gitlab.com/sleberknight</url>
-        </developer>
-    </developers>
-
     <contributors>
         <contributor>
             <name>John Loizeaux</name>
             <url>https://gitlab.com/loizeaux</url>
         </contributor>
     </contributors>
 
-    <licenses>
```

Comment:
Why removing?

Reply:
This will now get pulled from kiwi-parent.